dynamic_analysis_pkg/dynamic_analysis_pkg/__init__.py

```diff
@@ -0,0 +1,167 @@
+
+from os import makedirs
+from os.path import join, exists, isdir, basename
+from shutil import move
+from datetime import datetime
+from tabulate import tabulate
+import pyunicore.client as unicore_client
+import pandas as pd
+
+UNICORE_ENDPOINT= 'https://bspsa.cineca.it/advanced/pizdaint/rest/core'
+
+
+class Results:
+
+    DEFAULT_CIRCUIT_DIR = '/home/data-bbp/20191017/'
+    DEFAULT_WD_BASE = '/home/simulation-results';
+    ORIGIN_CIRCUIT_PATH = '/store/hbp/ich002/antonel/O1/20191017/'
+    ORIGIN_DIR_PATH = '/scratch/snx3000/unicore/FILESPACE/'
+    job_base = join(UNICORE_ENDPOINT, 'jobs')
```

Comment:
This one is hard coded as well, so it should be upper case as well if I follow your own convention